content/lang/collections/traitattr/attributemining.en.php

```diff
@@ -0,0 +1,54 @@
+<?php
+/*
+ ------------------
+ Language: English
+ ------------------
+ */
+
+$LANG['ATTRI_MINING_TOOL'] = 'Attribute Mining Tool';
+$LANG['OCC_ATTRI_MINING_TOOL'] = 'Occurrence Attribute Mining Tool';
+$LANG['MUST_SELECT_TRAIT'] = 'You must select a trait';
+$LANG['MUST_SELECT_SOURCE_FIELD'] = 'A source field must be selected';
+$LANG['MUST_SELECT_FIELD_VALUE'] = 'You muct select at least one field value';
+$LANG['CHOOSE_ONE_STATE'] = 'Please choose at least one state to assign';
+$LANG['SELECT_COLLECT_TO_HARVEST'] = 'Select at last on collect to harvest from';
+$LANG['HOME'] = 'Home';
+$LANG['COLLECTION_MANAGEMENT'] = 'Collection Management';
+$LANG['ADJUST_COLLECTION_SELECTION'] = 'Adjust Collection Selection';
+$LANG['SEARCH_ALL_COLLECTION'] = 'Searching All Collections';
+$LANG['CLICK_DISPLAY_COLLEC_LIST'] = 'click to display collection list';
+$LANG['OCC_TRAITS_MAPPING'] = 'This module maps Occurrence Traits to specimens based on verbatium text field content.';
+$LANG['MORE'] = 'more';
+$LANG['PHENOLOGY_TRAIT_MAPPING'] = 'For instance, phenology traits can be coded in bulk by mapping various
+text strings displayed within Reproductive Condition text field to a controled phenology defined within the occurrence trait fields.
+Coded trait attributes can be downloaded and shared via the Darwin Core (DwC) Archive export and publishing tools.
+Traits are included within a ';
+$LANG['SEARCHING'] = 'Searching';
+$LANG['COLLECTION'] = 'Collections';
+$LANG['MEASUREMENT_OR_FACT'] = 'Measurement Or Fact';
+$LANG['DWC_EXTENSION_FILE'] = ' DwC Extension file.';
```

Comment:
I think you have too many spaces here, since there was a space in the HTML before this variable. Please check.